torch/nn/common_types.py

```diff
@@ -24,6 +24,15 @@
 _size_5_t = _scalar_or_tuple_5_t[int]
 _size_6_t = _scalar_or_tuple_6_t[int]
 
+# For arguments which represent optional size parameters (eg, adaptive pool parameters)
+_size_any_opt_t = _scalar_or_tuple_any_t[Optional[int]]
+_size_1_opt_t = _scalar_or_tuple_1_t[Optional[int]]
+_size_2_opt_t = _scalar_or_tuple_2_t[Optional[int]]
+_size_3_opt_t = _scalar_or_tuple_3_t[Optional[int]]
+_size_4_opt_t = _scalar_or_tuple_4_t[Optional[int]]
+_size_5_opt_t = _scalar_or_tuple_5_t[Optional[int]]
+_size_6_opt_t = _scalar_or_tuple_6_t[Optional[int]]
```

Comment:
Can you remove the unused types here (1/4/5/6)?

Reply:
Sure.